### ISSUE TITLE
Background Workers memory overflow

### ISSUE THREAD

**Issue by huifeideshui** [abpframework/abp#18036]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

According to the following addr https://docs.abp.io/en/abp/latest/Background-Workers  construction project, it is found that the memory primary key of the service increases and cannot be reduced when no code of business logic is written.

### Version

7.4

### User Interface

Common (Default)

### Database Provider

EF Core (Default)

### Tiered or separate authentication server

None (Default)

### Operation System

Windows (Default)

**Comment by maliming** [comment]
**Reproduction Steps**

**Comment by huifeideshui** [comment]
> **Reproduction Steps**  The project I set up as a document, writing nothing in the execution method Execute, still found a memory leak.

Whatever Volo. Abp. BackgroundWorkers. Quartz module implements, or AbpBackgroundWorkersHangfireModule, have this problem.

**Comment by realLiangshiwei** [comment]
I tested and ran this project for 12 hours, but no memory overflow problem was found.

https://github.com/344089386/AbpQuartzWorkerTest

<img width="790" alt="image" src="https://github.com/abpframework/abp/assets/16813853/15a20001-e454-4b58-9f55-aebbdd8010bb">

<img width="1131" alt="image" src="https://github.com/abpframework/abp/assets/16813853/028cc0b3-55c6-48f6-b955-44802cd17e54">

**Comment by 344089386** [comment]
Yes, I did not have a complete restoration of this Demo, so I deleted the reply and I will test it again
